src/grid/mod.rs

```diff
@@ -52,9 +52,23 @@ impl<T> Deref for Indexed<T> {
 
 impl<T: PartialEq> ::std::cmp::PartialEq for Grid<T> {
     fn eq(&self, other: &Self) -> bool {
+        // Compare raw grid content
+        // TODO: This is pretty inefficient but only used in tests
```

Comment:
is there a way to mark this with `#[cfg(test)]` or something?

Reply:
Yeah I think the whole impl can be marked with `#[cfg(test)]`.

Reply:
This will not work because ref-tests are not unit tests.

Reply:
Cool, it would be nice to mention why in the comments, for curious rustaceans :)